src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx

```diff
@@ -204,7 +214,9 @@ export const BridgeConfirm: FC<BridgeRouteProps & BridgeConfirmProps> = ({ confi
             <TimelineItem>
               <CustomRow>
                 <Row.Label>{buyAsset?.name ?? ''}</Row.Label>
-                <Row.Value>{bridgeAmountCryptoPrecision}</Row.Value>
+                <Row.Value>
+                  <Amount.Crypto symbol='' prefix='≈' value={bridgeAmountCryptoPrecision} />
```

Comment:
Same here 🙏🏽

Reply:
Fixed in [91d7519](https://github.com/shapeshift/web/pull/7241/commits/91d75198190530c4b54f25776c78f4efd02293ba)